### PR TITLE
Renaming the Class "Data"

### PR DESCRIPTION
**Pull request by omangbaheti**
Minor Nitpick, I know

### CODE REVIEW

File: Assets/Scripts/ViconNexusUnityStream/CustomHandScript.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ public class CustomHandScript : CustomSubjectScript
         protected string marker_FA1 = "FA1";
         protected string marker_WRA = "WRA";
         protected string marker_WRB = "WRB";
-    
+
         // thumb markers
```
Comment (ahmed-shariff) [inline Assets/Scripts/ViconNexusUnityStream/CustomHandScript.cs:69]: remove these empty lines.

File: Assets/Scripts/ViconNexusUnityStream/CustomSubjectScript.cs [hunk 2]
```diff
@@ -254,60 +254,14 @@ void ProcessData(Data data, string text)
                     {
                         SetPreviousData(marker, _data);
                     }
-                    data.data[marker] = _data;
-                }
-
-                if (gapFillingStrategy == GapFillingStrategy.FillRelative && !dataValid && segment.Value.Count > 1)
```
Comment (ahmed-shariff) [inline Assets/Scripts/ViconNexusUnityStream/CustomSubjectScript.cs:260]: why is this removed?

File: Assets/Scripts/ViconNexusUnityStream/CustomHWDScript.cs [hunk 3]
```diff
@@ -54,13 +54,13 @@ private void OnValidate()
         {
             if (Application.isPlaying)
             {
-                rotFilter = new OneEuroFilter<Quaternion>(90, rotFilterMinCutoff, rotFilterBeta);
-                posFilter = new OneEuroFilter<Vector3>(90, posFilterMinCutoff, posFilterBeta);
+                rotFilter = new(90, rotFilterMinCutoff, rotFilterBeta);
+                posFilter = new(90, posFilterMinCutoff, posFilterBeta);
```
Comment (ahmed-shariff) [inline Assets/Scripts/ViconNexusUnityStream/CustomHWDScript.cs:58]: Since the variables are not defined here, I would fully qualify the object creation instead of using the short form.

**Review by ahmed-shariff — CHANGES_REQUESTED** [review]
Lots of empty line changes, do remove them. See comments for other changes.

File: Assets/Scripts/ViconNexusUnityStream/CustomSubjectScript.cs (hunk 2 above)
Comment (omangbaheti) [reply Assets/Scripts/ViconNexusUnityStream/CustomSubjectScript.cs:260]: o wait this shouldnt have happened. Maybe this has some changes leftover from something else that i was trying

**Review by ahmed-shariff — APPROVED** [review]
